bin/nvh

```diff
@@ -606,9 +606,14 @@ function can_use_xz() {
   if [[ "${uname_s}" = "Linux" ]] && command -v xz &> /dev/null ; then
     # tar on linux is likely to support xv if it is available separately
```

Comment:
Unrelated to my PR, but I noticed this typo ("xv") still exists here.

"xv" --> "xz"

---

`tj/n` has this comment now:

`# tar on linux is likely to support xz if it is available as a command`